pdk/build_defs.bzl

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Providers for PDKs to be used by downstream synthesis.
+"""
+
+StandardCellInfo = provider("Contains information about the standard cells used for synthesis", fields = ["corners", "default_corner"])
+CornerInfo = provider("Contains information about the standard cells used for synthesis", fields = ["liberty", "ccsnoise", "leakage", "corner_name"])
```

Comment:
If these are intended to be reusable I think it could help to document what these fields are, at least what type they have. (Are ccsnoise and leakage booleans here too?)